virttest/utils_disk.py

```diff
@@ -600,6 +601,88 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def resize_xfs_filesystem(session, partition, fstype='xfs', size=None):
```

Comment:
The `size` can be a positive or negative number, otherwise, this function is only used for 'enlarging', so the default should be `0` instead of `None`, or it doesn't make sense to have a default value.